charts/ratify/values.yaml

```diff
@@ -14,6 +14,16 @@ cosignKeys: []
 
 notation:
   enabled: true
+  trustPolicies:
+    - name: default
+      registryScopes:
+        - "*"
+      signatureVerification:
+        level: strict
+      trustStores:
+        - ca: certs
```

Comment:
How can different trust policies use different stores?

Reply:
> How can different trust policies use different stores?

Consider the following verifier.yaml:
```
apiVersion: config.ratify.deislabs.io/v1beta1
kind: Verifier
metadata:
  name: verifier-notation
spec:
  name: notation
  artifactTypes: application/vnd.cncf.notary.signature
  parameters:
    verificationCertStores:
      my_private_cert:
        - kmpprovider-akv
      my_public_cert:
        - inline_mar_images
    trustPolicyDoc:
      version: "1.0"
      trustPolicies:
        - name: my_private_registry
          registryScopes:
            - "skalantari_registry.azurecr.io/*"
          signatureVerification:
            level: strict
          trustStores:
            - ca:my_private_cert
          trustedIdentities:
            - "my identity"
        - name: mar_images
          registryScopes:
            - "mcr.azurecr.io/*"
          signatureVerification:
            level: strict
          trustStores:
            - ca:inline_mar_images
          trustedIdentities:
            - "mar identity"
```
It defines two trustPolicies, each include their trustStores, trustedIdentities, and registryScopes. In my PR, I moved this section to values.yaml so that the user can configure it before the installation. Note that [verificationCertStores](https://github.com/ratify-project/ratify/blob/252afd825c208e976582a7b2ce8ff81ee2533f57/charts/ratify/templates/verifier.yaml#L15C1-L15C28) is filled with the values specified in values.yaml too (notationCerts in values.yaml is used)

Reply:
Can you describe how users configure values.yaml for the verifier.yaml example?

Reply:
I think what @yizha1 is getting is how would the user from the helm chart be able to set up the `verificationCertStores` field because that would require the mapping between the notation trust store name and the ratify KMP names that belong to that trust store.

Reply:
discussed in PR review meeting, customer will need provide the 3 piece info below
helm install gatekeeper/gatekeeper  \
--name-template=gatekeeper \
--namespace gatekeeper-system --create-namespace \
--set trustpolicies[0].registryScopes[0] = "skalantari_registry.azurecr.io/*" \
--set trustpolicies[0].trustedIdentities[0]= "cert identity*" \
--set trustpolicies[0].notationCerts[0]= "file content"

Reply:
@yizha1 , helm chart tsa support is out of scope for this PR

Reply:
discussion from PR review, @shahramk64  will continue to explore the --set commands. 
- follow up with Yi, is it a goal  for customer provide least input

Reply:
just a note, cosign also support trust policies,  please evalaute if these are application for cosign. If they are not application, we need to make sure they have notation in the name

Reply:
@susanshi why the tsa support is out of scope? It was not exposed previously for timestamping feature, but it is required especially for production.

Reply:
> discussion from PR review, @shahramk64 will continue to explore the --set commands.
> 
> * follow up with Yi, is it a goal  for customer provide least input

Only the necessary parameters should be exposed. For example, the parameter of verificdation level is not needed at this moment unless users want it.